AlphaWallet/Settings/Models/AddCustomChain.swift

```diff
@@ -8,7 +8,7 @@ import PromiseKit
 enum AddCustomChainError: Error {
     case cancelled
     case others(String)
-
+    
```

Comment:
Can you avoid making such whitespace changes unless they are intentional (makes diffs hard to read)? We generally keep empty lines without white spaces.